rosstalk.js

```diff
@@ -41,7 +41,7 @@ instance.prototype.init_tcp = function() {
 	}
 
 	if (self.config.host) {
-		self.socket = new tcp(self.config.host, 7788);
+		self.socket = new tcp(self.config.host, self.config.port);
```

Comment:
Since the port config is being added, even though there's a default, you'll need to test if it exists prior to creating the socket:
```
if (self.config.port === undefined) {
	self.config.port = 7788;
}
```